frontend/js/components/TitleEditor.vue

```diff
@@ -106,8 +114,13 @@
         const titleValue = typeof title === 'string' ? title : title[this.currentLocale.value]
         return titleValue || this.warningMessage
       },
+      customlink: function () {
+        const localizedCustomPermalink = this.localizedCustomPermalink.length > 0 ? JSON.parse(this.localizedCustomPermalink) : {}
+        return Object.keys(localizedCustomPermalink).length > 0 ? localizedCustomPermalink[this.currentLocale.value] : (this.customPermalink ? this.customPermalink : false)
+      },
       permalink: function () {
-        return this.fieldValueByName('slug')[this.currentLocale.value]
+        const customLocalizedPermalinkbase = this.customLocalizedPermalinkbase.length > 0 ? JSON.parse(this.customLocalizedPermalinkbase) : {}
+        return Object.keys(customLocalizedPermalinkbase).length > 0 ? customLocalizedPermalinkbase[this.currentLocale.value].concat(this.fieldValueByName('slug')[this.currentLocale.value]) : this.fieldValueByName('slug')[this.currentLocale.value]
```

Comment:
I think there is no fallback for undefined locales in`getLocalizedPermalinkBase()`. For example, this would crash on the frontend:
```
protected function getLocalizedPermalinkBase()
    {
        return [
            'en' => 'content',
        ];
    }
}
```
when a user selects `FR` in the CMS. However `localizedCustomPermalink` seems to fallback nicely to the basic value of `$permalinkBase` for missing locales.

Reply:
@pboivin . Fixed to fall back to the records slug with the assumption that the permalinkBase for the undefined locale has not been set. This is also limited by how the FE constructs the full URL from the base url (Cannot fall back to the basic value of `$permalinkBase` since it would be missing from the base URL). See https://github.com/area17/twill/pull/1092#discussion_r741870901

Reply:
Thanks! I think The fallback to the bare slug is a good alternative.